### PR TITLE
Load "small" before "full" report

### PR DESCRIPTION
**Pull request by swaterkamp**
**Checklist**:

- [X] Tests
- [X] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: gsa/src/web/pages/reports/reportentitiescontainer.js [hunk 1]
```diff
@@ -181,8 +183,8 @@ class ReportEntitiesContainer extends React.Component {
 
 ReportEntitiesContainer.propTypes = {
   children: PropTypes.func,
-  counts: PropTypes.counts.isRequired,
-  entities: PropTypes.array.isRequired,
+  counts: PropTypes.oneOfType([PropTypes.counts, PropTypes.object]).isRequired,
```
[the remark this answers is not in the document]
Comment (swaterkamp) [reply gsa/src/web/pages/reports/reportentitiescontainer.js]: I need an object instead of counts, because if I pass a new CollectionCount() its construction sets all fields to 0 resulting in faulty information on the detailspage (it will tell me that the report is empty instead of showing Loading).

[the remark this answers is not in the document]
Comment (bjoernricks) [reply gsa/src/web/pages/reports/reportentitiescontainer.js]: Why not passing undefined instead? undefined has a clear meaning.

[the remark this answers is not in the document]
Comment (swaterkamp) [reply gsa/src/web/pages/reports/reportentitiescontainer.js]: For example in ResultsTab I would have to check again, if it's undefined when calling counts.filtered or counts.all. in order not to have to add several tests like that i just pass an empty object.

[the remark this answers is not in the document]
Comment (bjoernricks) [reply gsa/src/web/pages/reports/reportentitiescontainer.js]: you could init the counts instead when passing undefined
```
{counts = {}} = props
```
Seems to be much cleaner to me :-)

[the remark this answers is not in the document]
Comment (swaterkamp) [reply gsa/src/web/pages/reports/reportentitiescontainer.js]: In ReportEntitiesContainer  i would still have to accept CollectionCount and Object then.

[the remark this answers is not in the document]
Comment (bjoernricks) [reply gsa/src/web/pages/reports/reportentitiescontainer.js]: ok. i'll take a look after the pr is merged.
